sympy/matrices/matrices.py

```diff
@@ -4366,7 +4368,11 @@ def norm(self, ord=None):
 
         # Matrix Norms
         else:
-            if ord == 2:  # Spectral Norm
+            if ord == 1:  # Maximum column sum
+                return Max(*[sum(self.col(i).applyfunc(abs))
```

Comment:
I would prefer having `applyfunc` called only once (for the whole matrix) instead of separately for each column.